### ISSUE TITLE
[DUPE of 7122] Item counter accessibility with no label

### ISSUE THREAD

**Issue by pyrmont-dev** [SAP/spartacus#10734]
This component fails accessibility test as there is no label around the input for screen readers.

Adding an aria-labelledby attribute to pass a label value would fix the issue.

`<cx-item-counter [ariaLabelledby]="Example label">`

https://github.com/SAP/spartacus/blob/77452a300081d84b57f96fd8c199fb6862bfd871/projects/storefrontlib/src/shared/components/item-counter/item-counter.component.html#L10

**Comment by saad-mo** [comment]
done with #7122 (PR #12702)

**Comment by KateChuen** [comment]
Tested in https://github.com/SAP/spartacus/issues/7122
